crates/l2/contracts/src/l1/CommonBridge.sol

```diff
@@ -118,7 +118,26 @@ contract CommonBridge is
         while (startingGas - gasleft() < amount) {}
     }
 
-    function _sendToL2(address from, SendValues memory sendValues) private {
+
+    bytes3 internal constant EIP7702_PREFIX = 0xef0100;
+    uint160 internal constant ADDRESS_ALIASING = uint160(0x1111000000000000000000000000000000001111);
+    function _getAddressAlias(address from) private view returns (address) {
```

Comment:
Let's add a comment saying the idea was inspired by Optimism. Also, please add some documentation explaining how this works at a high-level.